### ISSUE TITLE
histbn - warning

### ISSUE THREAD

**Issue by bnemati** [bnemati/CAO1#1]
line 99 gives error messages like:
Warning: Stretching right bound from 0.271 to 0.280 to match bin width. 

This is useful but there is too many times this happens. I'd like to change the behavior so that this condition is signified by some marking on the plot, more 'quietly'. One possibility is a '>' symbol on the right side of the plot window to indicate the stretching.

**Comment by samhmiller** [comment]
I'll look at this today. In the meantime, if you need to suppress the warning, type histbn(...,'suppressWarning',True).

**Comment by bnemati** [comment]
OK that's a good feature already. Also, I'm thinking that the [matlabutilities](https://github.com/bnemati/CAO1/tree/master/matlabUtilities) folder should be the one where all the latest and greatest versions of the utilities should be placed. We can then place copies in other folders if we want.

**Comment by samhmiller** [comment]
Good idea, I’ll do that and then do a quick search through the repository to make sure there aren’t any outdated versions being used (or if they are we can just take note of them)

**Comment by bnemati** [comment]
Thanks!

Sent from my iPad

On Mar 28, 2019, at 11:33 AM, samhmiller <notifications@github.com> wrote:

Good idea, I’ll do that and then do a quick search through the repository to make sure there aren’t any outdated versions being used (or if they are we can just take note of them)
